src/parser/jobs/gnb/modules/Ammo.tsx

```diff
@@ -170,6 +175,18 @@ export default class Ammo extends Module {
 			value: this.erroneousCircles,
 		}))
 
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.BLOODFEST.icon,
+			content: <Trans id="gnb.ammo.leftover-ammo.content">
+				Avoid having leftover ammo at the end of a fight, consider using the ammo earlier if possible.
```

Comment:
Might be worth adding a brief _why_. Why should I avoid having ammo left over?

Reply:
Expanded on this, Burst Strike > Any GCD in the standard combo. Ammo combo is still better than it though.